.dockerignore

```diff
@@ -12,4 +12,7 @@ devops
 cover-existing.html
 coverage-existing.txt
 report-existing.xml
-testlogs-existing.txt
\ No newline at end of file
+testlogs-existing.txt
+hack/generator
```

Comment:
This doesn't seem right?

Reply:
oh wait it's dockerignore not .gitignore

Reply:
Driveby change - I noticed that building the image was sending 1GB of stuff to docker before ignoring most of it.